aiida/manage/configuration/options.py

```diff
@@ -71,7 +71,14 @@ def validate(self, value: Any) -> Any:
                 GlobalOptionsSchema.model_construct(), attribute, value
             )
         except ValidationError as exception:
-            raise ConfigurationError(str(exception)) from exception
+            messages = []
+            for error in exception.errors():
+                try:
+                    messages.append(str(error['ctx']['error']))
+                except KeyError:
+                    messages.append(f"Invalid value for `{error['loc'][0]}`: {error['msg']}")
+
```

Comment:
Not sure I understand why this is needed for this implementation.

Reply:
That's just an implementation detail of `pydantic`. I amn not a 100% sure when the structure of their `exception.errors()` changes, but I need to manually reconstruct the error message if I want it to be nicely formatted. The default pydantic formatting is not really suitable.

Reply:
Okay, then I choose to simply trust you :sunglasses:

Reply:
Bad move, my friend, bad move ! ;)